doc/api.yml

```diff
@@ -276,6 +276,12 @@ paths:
           description: "Filter students by course they are linked to"
           schema:
             type: string
+        - name: student_status
```

Comment:
status is enough.
Be consistent in your naming, as "last_name, first_name, ref" aren't prefixed with "student", then there's no need to.

Also, prefixing is mostly used to avoid confusion, there is no possible confusion here

Reply:
checked